app/assets/javascripts/discourse/app/components/sidebar/messages-section.js

```diff
@@ -2,13 +2,14 @@ import { cached } from "@glimmer/tracking";
 
 import { getOwner } from "discourse-common/lib/get-owner";
 import GlimmerComponent from "discourse/components/glimmer";
+import { bind } from "discourse-common/utils/decorators";
 import GroupMessageSectionLink from "discourse/lib/sidebar/messages-section/group-message-section-link";
 import PersonalMessageSectionLink from "discourse/lib/sidebar/messages-section/personal-message-section-link";
 
 export const INBOX = "inbox";
-const UNREAD = "unread";
+export const UNREAD = "unread";
 const SENT = "sent";
-const NEW = "new";
+export const NEW = "new";
 const ARCHIVE = "archive";
```

Comment:
Btw, you seemed to put these in an array in a previous PR: https://github.com/discourse/discourse/pull/17106/files, why not do the same here?